MyBlog.Services/Areas/Users/UserService.cs

```diff
@@ -37,8 +37,21 @@ public async Task<UserDto> GetByIdAsync(int id)
         return userDto;
     }
 
+    public async Task<User> GetByEmailAsync(string email)
+    {
+        var user = await _context.Users.FirstOrDefaultAsync(u => u.Email == email)
+                   ?? throw new NotFoundException($"User with Email: {email} is not fount");
+        
+        return user;
+    }
+
     public async Task<int> CreateAsync(UserDtoInput userInput)
     {
+        if (_context.Users.Any(u => u.Email == userInput.Email || u.Phone == userInput.Phone))
+        {
+            throw new ExistsException($"User with Email: {userInput.Email} or phone: {userInput.Phone} exists");
+        }
+        
```

Comment:
it is good, that you improve new custom Exception, but i think BadRequest is better, because you write API, and imagine, a lot of people will be send requests to your api, and they all know about BadRequest, it is standart, but what about ExistsException ?They are new about that
also, you can remove {userInput.Email} and {userInput.Phone} - user know, if it`s wrong, that will be email or password